frontend/src/index.scss

```diff
@@ -221,9 +221,9 @@ h3,
 h4,
 h5,
 h6 {
-	font-family: var(--header-font-family) !important;
-	margin: 0 !important;
-	color: var(--text-primary) !important;
+	font-family: var(--header-font-family);
+	margin: 0;
+	color: var(--text-primary);
```

Comment:
Was competing with some of the styles we wanted to apply in our `Heading` component.